rdkcentral/Lightning-UI-Components
# feat(Toggle): add Toggle element

This PR adds the Toggle element as well as the related docs, stories, and tests. 

### Testing

- `git checkout feat/add-toggle`
- `npm run test`
- `npm start`
- navigate to `elements > Toggle`
    - toggle the `checked` Control, the toggle should switch between a `checked`(green) and `unchecked`(grey) state

 ### Checklist
